site/en/tutorials/quickstart/beginner.ipynb

```diff
@@ -339,14 +440,53 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 13,
       "metadata": {
         "id": "cnqOZtUp1YR_"
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "<tf.Tensor: shape=(5, 10), dtype=float32, numpy=\n",
+              "array([[1.0304422e-08, 1.9711484e-08, 9.6276726e-06, 1.5255770e-04,\n",
+              "        3.2729534e-11, 2.4521808e-07, 4.5261253e-14, 9.9983692e-01,\n",
+              "        1.1976965e-07, 4.1715063e-07],\n",
+              "       [6.4603672e-10, 2.1719732e-05, 9.9996853e-01, 8.6020473e-06,\n",
+              "        1.4522603e-17, 1.0268549e-06, 3.8995367e-08, 4.8680898e-15,\n",
+              "        6.9020516e-08, 1.5862728e-14],\n",
+              "       [1.7178706e-07, 9.9939167e-01, 4.1672323e-05, 1.2468613e-06,\n",
+              "        1.3816109e-05, 2.3295547e-06, 4.7004680e-05, 4.3922896e-04,\n",
+              "        6.2010091e-05, 7.9445419e-07],\n",
+              "       [9.9267608e-01, 3.2803295e-05, 1.3152238e-03, 3.2566506e-06,\n",
+              "        3.9083403e-05, 3.8559776e-04, 1.6294547e-03, 3.8296464e-03,\n",
+              "        3.1000513e-05, 5.7887370e-05],\n",
+              "       [1.0925884e-06, 1.0096958e-08, 1.2631187e-05, 2.5030150e-07,\n",
+              "        9.8898894e-01, 1.1879993e-06, 1.4527029e-05, 4.4708585e-04,\n",
+              "        5.5869390e-07, 1.0533878e-02]], dtype=float32)>"
+            ]
+          },
+          "execution_count": 13,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
       "source": [
         "probability_model(x_test[:5])"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-47O6_GLdRuT"
+      },
+      "source": [
+        "## Conclusion\n",
+        "\n",
+        "Congratulations! You have trained and run a machine learning model using a prebuilt data set using the [Keras](https://www.tensorflow.org/guide/keras/overview) API.\n",
+        "\n",
+        "To learn more about using Keras, checkout the tutorials on [image classificnation](https://www.tensorflow.org/tutorials/keras/classification) or [text classification](https://www.tensorflow.org/tutorials/keras/text_classification). If you want learn more about loading and preparing data, see the tutorials on [image data loading](https://www.tensorflow.org/tutorials/load_data/images) or [CSV data loading](https://www.tensorflow.org/tutorials/load_data/csv).\n"
```

Comment:
Nit: typo (should be "classification")

Reply:
For more accessibility-friendly language, I'd recommend "see the tutorial" -> "visit"/"refer to"/"go to"  ["the tutorial"]

Reply:
Nit: I think it's "check out" (<- "checkout")